src/Memento.jl

```diff
@@ -60,9 +61,22 @@ const _loggers = Dict{AbstractString, Logger}(
 
 const LOGGER = getlogger(@__MODULE__)
 
+# Cached values either to lazily load or precompile.
+const _default_formatter = DefaultFormatter(DEFAULT_FMT_STRING)
+const _localtz = Ref{Union{Dates.TimeZone, Nothing}}(nothing)
+
+function getlocalzone()
+    if _localtz[] === nothing
+        global _localtz[] = localzone()
+    end
+
+    return _localtz[]
+end
+
 function __init__()
     Memento.config!(DEFAULT_LOG_LEVEL)
     Memento.register(LOGGER)
 end
 
+_precompile_()
```

Comment:
Is this the right way to do this now?

Reply:
That's what's documented in SnoopCompiler.jl and it does seem to make a difference in load times.

Reply:
Fair enough.
TIL